docs-src/pages/UiTabs.vue

```diff
@@ -66,18 +64,22 @@
             <h4 class="page__demo-title">Custom header with HTML</h4>
 
             <ui-tabs fullwidth>
-                <ui-tab :key="tab.title" v-for="tab in demoTabs">
+                <ui-tab :key="tab.title" v-for="tab in demoTabs" :icon="tab.icon">
```

Comment:
Since we use the header slot here, which replaces the icon, we can omit the icon prop here right?